tox.ini

```diff
@@ -18,14 +18,9 @@ envlist =
 
 [testenv]
 commands =
-    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof -ra {env:_PYTEST_TEST_OPTS:} {posargs:testing}
-    coverage: coverage report -m --skip-covered
-setenv =
-    coverage: _PYTEST_TOX_COVERAGE_RUN=coverage run -m
-    coverage: _PYTEST_TOX_EXTRA_DEP=coverage-enable-subprocess
-    coverage: COVERAGE_FILE={toxinidir}/.coverage
-    coverage: COVERAGE_PROCESS_START={toxinidir}/.coveragerc
+    {env:_PYTEST_TOX_COVERAGE_RUN:} pytest --lsof -ra {posargs:testing}
```

Comment:
Please keep the "coverage" factor?!

Reply:
Unless I'm missing something, the "coverage" factor will only be considered when executing `tox -e *coverage*` no? If that's the case, we don't execute any test env with "coverage" in the name, that's why I thought this could be removed.

Reply:
Not on CI, but you can do `tox -e py36-coverage` manually.

Reply:
Oh I see, did not thought about that use case. Will add it back. 👍